store/instance.go

```diff
@@ -581,6 +581,12 @@ func findInstanceQuery(find *api.InstanceFind) (string, []interface{}) {
 	if v := find.EnvironmentID; v != nil {
 		where, args = append(where, fmt.Sprintf("environment_id = $%d", len(args)+1)), append(args, *v)
 	}
+	if v := find.Host; v != nil {
```

Comment:
%s and port is string too I think.

Reply:
I guess you make a mistake? The code here is used to build the SQL query like `host = $1`

Reply:
My bad